### ISSUE TITLE
0.7.0 version crashes Outlook 2021 while creating new Jitsi meeting

### ISSUE THREAD

**Issue by garett80** [timetheoretical/jitsi-meet-outlook#60]
Hello, faced with trouble while using new (0.7.0) plugin version - it crashes my Outlook 2021 (Home & Business).

**Comment by garett80** [comment]
No, just trying to create new meeting and Outlook suddenly crashes. This is only problem with 0.7.0, 0.6.6 is normal.

**Comment by timetheoretical** [comment]
Hm, i can't reproduce this in my Outlook 2019 instance

**Comment by garett80** [comment]
So you think this is only Office (Outlook 2021) problem? Is there any updates planning?

**Comment by Labi4** [comment]
interesting thing with Outlook 2021 Home&Business in case if you will change config file with wrong parameters outlook addon will disappear from panel. and there is no way to get it back. remove & install is not working.

**Comment by andreasn82** [comment]
Same here. Outlook crashes when we try to insert the Jitsi URL in an already created meeting. If we hit the "New Jitsi Meeting" in the Outlook Calendar it works fine.

**Comment by andreasn82** [comment]
Same issue here with Outlook 2021 - Creating a new meeting "from scratch" with the button in Outlook works fine, but adding meeting information by hitting the button _inside_ a meeting makes Outlook crash.
